pygmt/helpers/decorators.py

```diff
@@ -36,6 +36,18 @@
         color : str
             Select color or pattern for filling of symbols or polygons. Default
             is no fill.""",
+    "V": """\
+        verbose : str
```

Comment:
FYI, the GMT experimental long options use `verbosity` and `spherical` for **-V** and **-j** (https://github.com/GenericMappingTools/gmt/blob/master/src/gmt_init.c#L383), while GMT.jl uses `verbose` and `spheric_dist | spherical_dist`(https://www.generic-mapping-tools.org/GMT.jl/latest/#GMT.grdmask).

Reply:
I would definitely prefer 'verbose' for **V**, it's similar to most command-line arguments.

As for 'j', I've merged the PR already, but could make a quick change. The GMT.jl documentation actually points to http://docs.generic-mapping-tools.org/latest/gmt.html#j-full (i.e. capital **J**) rather than the proper link at https://docs.generic-mapping-tools.org/latest/gmt.html#distcalc-full (notice the `distcalc` in the link).

Reply:
Yes, I also prefer `verbose` for **-V**. 

For **j**, I'm OK with the names. It depends on if we want to keep consistent with other GMT projects.

Reply:
FYI, I believe the GMT server is up.